co-log/tutorials/1-intro/Intro.md

```diff
@@ -42,23 +37,13 @@ example0 = do
     putStrLn "Example 0: Second message"
 ```
 
-Using `putStrLn` for logging is a very simple and basic approach for logging.
-When your application becomes bigger and more complex, you might want to bring
-some logging library into it. For example, you might want to do something from
-the following list:
+Using `putStrLn` for logging is a very simple and basic approach for logging.  When your application grows bigger and more complex, you might want to introduce some logging library to it. For example, you might want to do something from the following list:
 
-1. Specify messages with the given `Severity` so you can control the verbosity
-   of the output.
-2. Automatically print timestamps, thread ids, source code line of the logging
-   with each message.
-3. You would like to submit some statistics to some web-server with each logging
-   message so later you can have analytics provided by external parties.
+1. Specify messages with a given `Severity` so you can control the verbosity of the output.
+2. Automatically print timestamps, thread IDs, and source code lines of the log messages.
+3. Submit some statistics to a web server along each log message so later you can have analytics provided by external parties.
 
-Now let's look at how you can use `LogAction` instead of `putStrLn` to achieve the
-same goal. With `co-log` you need to have a value of type `LogAction` that defines
-how you are going to do logging. So you configure your logging settings separately and
-then pass and use this `LogAction` value. See the following example for more
-details:
+Now, let's look at how you can use `LogAction` instead of `putStrLn` to achieve the same goal. With `co-log`, you need to have a value of type `LogAction` that defines how you are going to perform logging. So you configure your logging settings separately and then pass and use this `LogAction` value. See the following example for more details:
```

Comment:
Each long sentence was broken over multiple lines to fit into 80 char limit and to see exact diffs on GitHub. Since the lines are collapsed now, I don't see exact diffs on GitHub. No need to rewrite it back, I'm just teling why it was done like this in the first place :slightly_smiling_face: